ddp-workspace/projects/ddp-osteo/src/app/components/welcome/welcome.component.html

```diff
@@ -209,7 +209,7 @@ <h2 class="social-media-header__text" translate>
             <div class="feeds">
                 <div class="twitter-feed">
                     <p class="big bold" translate>HomePage.SocialMediaSection.TwitterFeedTitle</p>
-                    <app-twitter-timeline-widget></app-twitter-timeline-widget>
+                    <toolkit-twitter-timeline-widget></toolkit-twitter-timeline-widget>
```

Comment:
`twitter-timeline-widget` moved into the Toolkit because we can use it in Osteo and new Brain.